src/storage-types/src/controller.rs

```diff
@@ -327,14 +325,25 @@ impl std::str::FromStr for PersistTxnTablesImpl {
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         match s {
-            "off" => Ok(PersistTxnTablesImpl::Off),
-            "eager" => Ok(PersistTxnTablesImpl::Eager),
+            "off" | "eager" => Ok(PersistTxnTablesImpl::Eager),
```

Comment:
I _think_ we'd never see "off" here (or 0 below), but it's easy to handle them, so figured I may as well